.github/workflows/build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Node Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 14
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14
+    - run: npm install
+    - run: npm run build
+      env:
+        GATSBY_SEGMENT_WRITE_KEY: ${{ secrets.GATSBY_SEGMENT_WRITE_KEY }}
+        GATSBY_TRACK_PAGE: true
+        GATSBY_DRIFT: ${{ secrets.GATSBY_DRIFT }}
```

Comment:
what is this?

Reply:
Segment key and all, need to provide at build time

Reply:
i mean `GATSBY_DRIFT`

Reply:
Will fix that